run.sh

```diff
@@ -4,5 +4,15 @@ set -e
 
 TAG="vw-reservation:latest"
 
+# Make a random name for this container so we can delete it later
+NAME="VW_ID4_$(date +%s)"
+
+# Build and run container
 docker build -f Dockerfile . -t "$TAG"
-docker run -e USERNAME -e PASSWORD -e PAGE_LOAD_TIMEOUT_SECONDS "$TAG"
+docker run --name "$NAME" --env-file=.env.template "$TAG"
```

Comment:
When I run this locally my configuration is incorrect because we are assuming the name of the `env` file. Can we instead be explicit about the environment vars we are passing by using `-e`? Then different people with different configuration approaches can use this a little easier

Similar comment to the `.ps1` script

Reply:
Will make that revert now.

I can't figure out how the env variables would be sourced without doing it manually.

Does this sound right:
1. Instruct user to copy .env.template into new .env file
2. Edit credentials
3. Source .env file into local machine env variables
4. Docker run in run.* files expects env variables to be set on local machine

For step 3, sounds like it should be fairly simple for unix to just expect and source the .env file?
Will need to look that up for powershell but it's probably fairly simple to source a file into env vars.

Reply:
> sourced without doing it manually

Different people will be using different config management. For example, when I use https://direnv.net/ it happens automatically. Other people will set it in their `.bashrc` and yet others will just run `export` before they need to run it. I'll merge this and add some more guidance in the readme now

Reply:
https://github.com/nmiodice/id4-reservation-tracker/pull/3/files